drivers/charger/charger_bq24190.c

```diff
@@ -8,7 +8,7 @@
 
 #include <errno.h>
 
-#include "bq24190.h"
+#include "charger_bq24190.h"
```

Comment:
I find this one redundant: `drivers/charger/charger_`

Reply:
So in case of new drivers, should we drop that prefix as well?

Reply:
I’d personally do, folder already _namespaces_ here

Reply:
Let's wait a while and find out others' opinions.

Reply:
I see and understand @gmarull's point here. If it is the norm, than this change should be withheld.

Reply:
Alright.